perfmetrics/scripts/testing_on_gke/examples/utils/utils.py

```diff
@@ -136,5 +143,190 @@ def unix_to_timestamp(unix_timestamp: int) -> str:
   return utc_timestamp_string
 
 
-def standard_timestamp(timestamp: int) -> str:
+def standard_timestamp(timestamp: str) -> str:
   return timestamp.split(".")[0].replace("T", " ") + " UTC"
+
+
+def timestamp_to_epoch(timestamp: str) -> int:
+  return int(
+      time.mktime(
+          time.strptime(
+              timestamp.split(".")[0].replace("T", " "), "%Y-%m-%d %H:%M:%S"
+          )
+      )
+  )
+
+
+def isRelevantMonitoringResult(
```

Comment:
Should this be renamed to is_relevant_monitoring_result in line with other functions?

Reply:
yes, I'll change it to `_is_relevant_monitoring_result` (_ prefix to indicate that this method is private to this file).

Reply:
done